### PR TITLE
task done

### PR DESCRIPTION
**Pull request by Rommelua**

### CODE REVIEW

File: src/main/java/com/dev/cinema/dao/impl/AbstractDao.java [hunk 1]
```diff
@@ -34,6 +39,9 @@ protected T add(T instance, Class<T> clazz) {
         try (Session session = HibernateUtil.getSessionFactory().openSession()) {
             Query<T> query = session.createQuery("from " + clazz.getSimpleName(), clazz);
             return query.getResultList();
+        } catch (Exception e) {
+            logger.error("Cant fetch all {} from DB", clazz.getName(), e);
```
Comment (dariiabutorina) [inline src/main/java/com/dev/cinema/dao/impl/AbstractDao.java:43]:
```suggestion
            logger.error("Can't create the {} instance", clazz.getName(), e);
```

Comment (dariiabutorina) [reply src/main/java/com/dev/cinema/dao/impl/AbstractDao.java:43]: I wouldn't understand what this message was about :)
Let's fix the message for all of the _**add**_ methods

**Review by dariiabutorina — COMMENTED** [review]
Overall, looks good. What about MainApp?

File: src/main/java/com/dev/cinema/dao/impl/AbstractDao.java (hunk 1 above)
Comment (Rommelua) [reply src/main/java/com/dev/cinema/dao/impl/AbstractDao.java:43]: This is getAll() method. I don't do logging in add() because of throwing exception.

**Comment by Rommelua** [comment]
> Overall, looks good. What about MainApp?

added!

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**